docs/src/user-guide/core-unstructured/clp.md

```diff
@@ -1,21 +1,10 @@
-# Using CLP for unstructured logs
+# clp
```

Comment:
Ditto should this be upper case?

Reply:
In this case, my goal is to refer more to the binaries rather than the category (or whatever you'd call it). This is what the sidebar looks like after the latest changes:

![image](https://github.com/y-scope/clp/assets/2454684/71415395-37a8-4428-a25b-ce10828bf4c1)

So I feel like it's more appropriate to keep clp and glt lowercase, similar to make-dictionaries-readable.

Reply:
That's fair then